lib/membrane_h264_plugin/parser.ex

```diff
@@ -57,7 +81,7 @@ defmodule Membrane.H264.Parser do
       any_of(
         %RemoteStream{type: :bytestream},
         %H264{alignment: alignment} when alignment in [:nalu, :au],
-        %H264.RemoteStream{alignment: alignment} when alignment in [:nalu, :au]
+        %H264{}
```

Comment:
Isn't it just an oversight?

Reply:
yes, we can have just `H264` since `:au` and `:nalu` are the only possible alignments